src/panels/lovelace/editor/hui-yaml-card-preview.ts

```diff
@@ -7,6 +7,11 @@ import createErrorCardConfig from "../common/create-error-card-config";
 import { HomeAssistant } from "../../../types";
 import { LovelaceCard } from "../types";
 
+interface ConfigValue {
```

Comment:
can you add `lovelace/editor/types.ts` and put this in, since it's shared between 2 files?

Reply:
Im not sure we need it in the edit card  dialog file. Is this not efficient `this._previewEl.value = { format: type, value: config };`